### DISCUSSION TITLE
现在已经完全覆盖李跳跳了吗？

### DISCUSSION THREAD

**Discussion by motian0828** [gkd-kit/gkd#77] [General]
可以只留gkd了吗？

**Comment by lisonge** [comment]
看个人意愿，李跳跳 是一款很优秀的软件，同时使用也是没问题的

**Reply by Chivent0506** [reply]
好像还没有自动查看原图，勾选原图的功能

**Reply by lisonge** [reply]
https://github.com/gkd-kit/subscription/issues/new/choose
